source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -38,13 +38,15 @@ open class AdaptiveCard {
 }
 
 public struct RenderConfig {
-    public static let `default` = RenderConfig(isDarkMode: false, buttonConfig: .default)
+    public static let `default` = RenderConfig(isDarkMode: false, buttonConfig: .default, supportsSchemaVersion1Dot3: false)
     let isDarkMode: Bool
     let buttonConfig: ButtonConfig
+	let supportsSchemaVersion1Dot3: Bool
```

Comment:
```suggestion
	let supportsSchemaVersion1_3: Bool
```

Reply:
Swiftlint only allows alphanumeric constants, not even underscore
Should I disable that property and rename?